src/VisualStudio/Core/Def/Implementation/Experimentation/EnhancedColorExperiment.cs

```diff
@@ -246,7 +227,7 @@ public void TrySetEnhancedColors(Guid themeId)
                     UpdateColorItem(colorItemMap, ClassificationTypeNames.OperatorOverloaded, DarkThemeMethodYellow);
                     UpdateColorItem(colorItemMap, ClassificationTypeNames.ControlKeyword, DarkThemeControlKeywordPurple);
                     UpdateColorItem(colorItemMap, ClassificationTypeNames.StructName, DarkThemeStructMint);
-                    UpdateColorItem(colorItemMap, ClassificationTypeNames.StaticSymbol, DefaultForegroundColor, DefaultBackgroundColor, isBold: true);
+                    UpdateColorItem(colorItemMap, ClassificationTypeNames.StaticSymbol, DefaultForegroundColor, DefaultBackgroundColor);
```

Comment:
❓ Why did the `isBold` value change here?

Reply:
We are looking for feedback that that bold statics are a desired default.

Reply:
+1 for this change from me, I hate bold statics :-).